src/Routes.tsx

```diff
@@ -23,9 +25,16 @@ const { BivariateManagerPage } = lazily(
 const initialUrl = new URL(localStorage.getItem('initialUrl') || '');
 
 export function RoutedApp() {
+  const [urlStore] = useAtom(urlStoreAtom);
```

Comment:
url store not used
should it be removed? 
if removed - atom should be started elsewhere (since you commented out ` // return forceRun(urlStoreAtom);` in RoutesWrap). Or you can just write `useAtom(urlStoreAtom);` without storing it

Reply:
will be refactored after upcoming app init refactor